src/components/vm/overview/bootOrder.tsx

```diff
@@ -312,7 +352,12 @@ class BootOrderModal extends React.Component {
             <Modal position="top" variant="medium" id={`${idPrefix}-window`} isOpen onClose={Dialogs.close} className='boot-order'>
                 <ModalHeader title={_("Change boot order")} />
                 <ModalBody>
-                    {this.state.dialogError && <ModalError dialogError={this.state.dialogError} dialogErrorDetail={this.state.dialogErrorDetail} />}
+                    {this.state.dialogError &&
+                        <ModalError
+                            dialogError={this.state.dialogError}
+                            {...this.state.dialogErrorDetail && { dialogErrorDetail: this.state.dialogErrorDetail } }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)